rules/manager_test.go

```diff
@@ -754,7 +754,7 @@ func TestUpdate(t *testing.T) {
 	defer tmpFile.Close()
 
 	err = ruleManager.Update(10*time.Second, []string{tmpFile.Name()}, nil)
-	testutil.Ok(t, err)
+	testutil.NotOk(t, err)
```

Comment:
`NotOk` since enabling the `KnownFiled(true)` will create an error for empty file due to `EOF`. Hence, here an error should occur. `ioutil.TempFile` will give an empty file and hence an error is expected.

Reply:
Not sure about this. Would break users.

Reply:
This is now back to `testutil.Ok`. :grinning:

Reply:
Resolving this since its now addressed.